package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-local-storage",
-  "version": "0.2.4",
```

Comment:
This is probably something the package maintainer should do and should be left out of PRs.